draft-ietf-scitt-architecture.md

```diff
@@ -182,9 +184,9 @@ To ensure readability, only a core set of terms is included in this section.
 
 **Editor's Note:**: *The label "394" is expected to be reserved by this document, in the COSE Header Parameters Registry.*
 
-The terms "header", "payload", and "to-be-signed bytes" are defined in {{RFC9052}}.
+The terms "header", "payload", and "to-be-signed bytes" are defined in {{-COSE}}.
 
-The terms claim is defined in {{RFC8392}}, and is repeated here for readability:
+The term "claim" is defined in {{RFC8392}} and is repeated here for readability:
```

Comment:
We also still have the term "Claim" in [the terminology section](https://ietf-wg-scitt.github.io/draft-ietf-scitt-architecture/x5chain-usage/draft-ietf-scitt-architecture.html#section-2). Should we not remove that and replace it with this wording? Because of his git diffs work I cannot recommend that here, or I would do it for you, but this seems like an important change to not cause confusion down the road.

Reply:
Agreed to remove claims in the terminology section.